packages/utils/src/namespaces.ts

```diff
@@ -63,3 +64,140 @@ export function getRequiredNamespacesFromNamespaces(
   }
   return required;
 }
+
+export function buildApprovedNamespaces(params: {
```

Comment:
Since this is user facing should we add the context you put in the PR description here too as a JSDoc and point out it’s relevant for wallets specifically?